app/frontend/test/unit/specs/aquifers/components/View.spec.js

```diff
@@ -12,79 +12,87 @@
     limitations under the License.
 */
 
-import { mount, createLocalVue } from '@vue/test-utils'
-import ViewComponent from '@/aquifers/components/View.vue'
-import Vuex from 'vuex'
-import axios from 'axios'
-import auth from '@/common/store/auth.js'
-import documentState from '@/common/store/documents.js'
-import aquiferCodes from '@/aquifers/store/codes'
-import VueRouter from 'vue-router'
+import { mount, createLocalVue } from "@vue/test-utils"
+import ViewComponent from "@/aquifers/components/View.vue"
+import Vuex from "vuex"
+import axios from "axios"
+import auth from "@/common/store/auth.js"
+import documentState from "@/common/store/documents.js"
+import aquiferCodes from "@/aquifers/store/codes"
+import VueRouter from "vue-router"
 
-jest.mock('axios')
+jest.mock("axios")
 
 const localVue = createLocalVue()
 localVue.use(Vuex)
 localVue.use(VueRouter)
 
 const aquiferFixture = {
-  aquifer_id: '4',
-  aquifer_name: 'Aggasiz',
-  location_description: 'Aggasiz/Sea Bird Island',
-  material_description: 'Sand and Gravel',
-  litho_stratographic_unit: 'Fraser River Sediments',
-  subtype_description: 'Unconfined sand and gravel - large river system',
-  vulnerability_description: 'High',
-  area: '63.3',
-  productivity_description: 'High',
-  demand: 'Low',
-  mapping_year: '1993'
+  aquifer_id: "4",
+  aquifer_name: "Aggasiz",
+  location_description: "Aggasiz/Sea Bird Island",
+  material_description: "Sand and Gravel",
+  litho_stratographic_unit: "Fraser River Sediments",
+  subtype_description: "Unconfined sand and gravel - large river system",
+  vulnerability_description: "High",
+  area: "63.3",
+  productivity_description: "High",
+  demand: "Low",
+  mapping_year: "1993",
+  resources: []
 }
 
-describe('View Component', () => {
-  const component = (options) => mount(ViewComponent, {
-    localVue,
-    router: new VueRouter(),
-    store: new Vuex.Store({
-      modules: { auth, aquiferCodes, documentState }
-    }),
-    stubs: ['aquifer-documents', 'aquifer-form'],
-    ...options
-  })
+describe("View Component", () => {
+  const component = options =>
+    mount(ViewComponent, {
+      localVue,
+      router: new VueRouter(),
+      store: new Vuex.Store({
+        modules: { auth, aquiferCodes, documentState }
+      }),
+      stubs: ["aquifer-documents", "aquifer-form"],
+      ...options
+    })
 
-  it('queries aquifer on load', () => {
+  it("queries aquifer on load", () => {
     const fetch = jest.fn()
 
     component({
       methods: {
         fetch,
-        fetchFiles () {
+        fetchFiles() {
           return {
             public: [],
             private: []
           }
+        },
+        fetchResourceSections() {
+          this.aquifer_resource_sections = []
         }
       }
     })
 
     expect(fetch).toHaveBeenCalled()
   })
 
-  describe('View mode', () => {
-    it('matches the snapshot', () => {
+  describe("View mode", () => {
+    it("matches the snapshot", () => {
```

Comment:
no big deal for now, but the linter rules in .eslintrc does the opposite (changes double quote to single quotes)